velero-plugins/common/types.go

```diff
@@ -23,3 +23,8 @@ const RestoreRegistryHostname string = "openshift.io/restore-registry-hostname"
 const MigrationRegistry string = "openshift.io/migration-registry"
 
 const SwingPVAnnotation string = "openshift.io/swing-pv"
+
+const StageAnnotation string = "openshift.io/stage-migration"
+const MigrateStagedAnnotation string = "openshift.io/migrate-staged"
```

Comment:
Having both "stage-migration" and "migrate-staged" seems like it might lead to confusion. Is there a clearer way of naming these?

Reply:
Yep, we can try to come up with something better. We know they're confusing. Too much stage. Too much migrate.